uart-protocol.md

```diff
@@ -388,15 +389,15 @@ Example, LEGO BOOST Color and Distance sensor:
       |     INFO_SI
       MESSAGE_INFO | LENGTH_8 | MODE_2
 
-#### INFO_SYMBOL
+#### INFO_UNITS
```

Comment:
I think "units" makes more sense than "symbol" but perhaps we should add a note that this is the same as `INFO_SYMBOL` in the ev3sources header files.